src/components/Modal.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { PropTypes } from 'prop-types';
+
+const MODAL_STYLES = {
+  position: 'fixed',
+  top: '50%',
+  left: '50%',
+  transform: 'translate(-50%, -50%)',
+  backgroundColor: '#fff',
+  padding: '50px',
+  zIndex: 1000,
+};
+
+const OVERLAY_STYLES = {
+  position: 'fixed',
+  top: '0',
+  left: '0',
+  right: '0',
+  bottom: '0',
+  backgroundColor: 'rgb(0, 0, 0, -7)',
+  zIndex: 1000,
+};
```

Comment:
- [ ] Why not add the style to a diff script? This could separate the code in a better way by following good practices. 🐱